var/spack/repos/builtin/packages/globus-toolkit/package.py

```diff
@@ -32,4 +32,7 @@ class GlobusToolkit(AutotoolsPackage):
     homepage = "http://toolkit.globus.org"
     url      = "http://toolkit.globus.org/ftppub/gt6/installers/src/globus_toolkit-6.0.1506371041.tar.gz"
 
+    version('6.0.1493989444', '9e9298b61d045e65732e12c9727ceaa8')
```

Comment:
Please add it below the current version, so that they remain sorted with the most recent at the top.

Reply:
Done